### PR TITLE
Join PiE Staff Page Rebuilt

### PR DESCRIPTION
**Pull request by kev-chien**
Join PiE Staff page has been rebuilt.

- [x] Now at /get-involved
- [x] "What We Do" section
- [x] "Staff Roles" section
- [x] "Become a Member" section

To be done:
- [ ] Work schedule
- [ ] Staff benefits
- [ ] Attach link to "pamphlet" (the why join us png)
- [ ] Style "What We Do" a little more (pictures, union/plus sign)

### CODE REVIEW

File: get-involved/index.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+    </div>
+</div>
+<div id="fall-row" class="rc-row vertical-align">
+    <div id="rc-video" class ="image-col embed-responsive embed-responsive-16by9">
+      <iframe class ="embed-responsive-item" src="https://www.youtube.com/embed/IQ-xtgo2VG4" allowfullscreen></iframe>
+    </div>
+    <div class="text-col text-left">
+        <div class="graphic-description">
+            <h3>COMPETITION</h3>
+            <p>Our capstone: an annual robotics competition for undeserving high school students in schools in the Bay Area. 10 years in the running. More than $30,000 in funding. Run completely by UC Berkeley students. Lots of robots.</p>
```
Comment (viviehn) [inline get-involved/index.md]: Can you change this to "More than $30,000 per year in funding"?

File: get-involved/index.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+hero-image: url(../assets/images/stock-images/pie-staff.jpg)
+hero-message: Get Involved!
+---
+
+<!-- WHAT WE DO -->
+<div class="container">
+    <h1 style="text-align: center">Join PiE Staff</h1>
+    <!--<h2>WHAT WE DO</h2>-->
+</div>
+<div class="rc-row vertical-align" >
```
Comment (viviehn) [inline get-involved/index.md]: I think this should also be wrapped in `<div class="container">` - it's currently taking up the full width of the page and looks a little jarring compared to everything else

File: get-involved/index.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+    <div class="text-col text-left" id="fall-row" >
+        <div class="graphic-description">
+            <h3>ENGINEERING</h3>
+            <p>We're mechanical engineers, software engineers, electrical engineers. We build, design, reiterate, and
```
Comment (viviehn) [inline get-involved/index.md]: I like this, but wondering if it's possible to also include something about the business, event planning, etc. side of PiE?

File: get-involved/index.md [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+    <div class="text-col text-right" id="spring-row" >
+        <div class="graphic-description">
+            <h3>OUTREACH</h3>
+            <p>We're UC Berkeley students who care about making a difference in the lives of underrepresented students in science and engineering.</p>
+            <h2><i class="fa fa-heart"></i> <i class="fa fa-users"></i> <i class="fa fa-book"></i></h2>
+        </div>
+    </div>
+</div>
+<div id="fall-row" class="rc-row vertical-align">
+    <div id="rc-video" class ="image-col embed-responsive embed-responsive-16by9">
```
Comment (viviehn) [inline get-involved/index.md]: The Engineering/Outreach section above this is split 50/50, but here, the video and the text next to it is not 50/50, and it looks a little off?

File: get-involved/index.md (hunk 1 above)
Comment (viviehn) [inline get-involved/index.md]: I really like this section!!!! 👍

File: get-involved/index.md (hunk 2 above)
Comment (viviehn) [inline get-involved/index.md]: Also, for these sections: Engineering/Outreach, Competition, PiE Staff Roles, Become a Member - would it be possible to have space in between each so it's not like one contiguous block?

File: about/about-us.html [hunk 5]
```diff
@@ -20,6 +20,7 @@ <h2 class="sub-heading">Our Robotics Competitions</h2>
 		</div>
 		<div id="fall-image" class="image-col"></div>
 	</div>
+	<!--
```
Comment (viviehn) [inline about/about-us.html]: Is there a reason you took out the spring section? I think we should keep this on there, as it's helpful for schools searching for information about our spring program

Comment (kev-chien) [inline about/about-us.html]: This was an accident! Good catch, will put back.

File: get-involved/index.md (hunk 1 above)
Comment (kev-chien) [inline get-involved/index.md]: Thank you :)

File: get-involved/index.md (hunk 2 above)
Comment (kev-chien) [inline get-involved/index.md]: addressed

File: get-involved/index.md (hunk 3 above)
Comment (kev-chien) [inline get-involved/index.md]: Wrote new description, see commit

File: get-involved/index.md (hunk 4 above)
Comment (kev-chien) [inline get-involved/index.md]: Addressed

File: get-involved/index.md (hunk 1 above)
Comment (kev-chien) [inline get-involved/index.md]: Addressed

**Comment by kev-chien** [comment]
Changes made! @viviehn

**Comment by viviehn** [comment]
Ah looks great Kevin! 💟

**Pull request merged**